docs/source/package.rst

```diff
@@ -0,0 +1,37 @@
+.. currentmodule:: torch.package
+
+torch.package
+=============
+
+API Reference
+-------------
+
+.. autoclass:: torch.package.PackageExporter
+
+  .. automethod:: __init__
```

Comment:
Is there a way to auto include all public methods? Might cause slow drift if you have to remember to add any public method to the API docs.